Wangscape/logging/Logging.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "AppenderType.h"
+#include "Level.h"
+#include "Logger.h"
+
+#include <ostream>
+
+#define LOG_ERROR getLogger(logging::Level::Error)
+#define LOG_WARNING getLogger(logging::Level::Warning)
+#define LOG_DEBUG getLogger(logging::Level::Debug)
+#define LOG_INFO getLogger(logging::Level::Info)
```

Comment:
I would prefer to avoid using `#define` unless it's absolutely necessary. Could these macros be replaced with functions or `static` (`const`?) instances/references?

I see that adding filenames and line numbers to messages would be easier with a `#define`, but that could be done with a `FILE_LINE` macro and `LOG_ERROR << FILE_LINE << ": Error message\n";`.

I don't think it would make sense if that was a standard part of `LOG_*`, or even `LOG_ERROR`. Many errors will be caused by faulty configurations, and the error message should direct attention to the code that needs fixing.